src/cookbook/images/network-image.md

```diff
@@ -31,15 +31,13 @@ Image.network(
     'https://docs.flutter.dev/assets/images/dash/dash-fainting.gif');
 ```
 
-## Placeholders and caching
+## Image fade in with placeholders
 
 The default `Image.network` constructor doesn't handle more advanced
-functionality, such as fading images in after loading, or caching images
-to the device after they're downloaded. To accomplish these tasks, see
-the following recipes:
+functionality, such as fading images in after loading. To accomplish this task, see
+the following recipe:
```

Comment:
Since this isn't a list anymore, you can just inline the mention.

```suggestion
functionality, such as fading images in after loading.
To accomplish this task,
check out [Fade in images with a placeholder][].
```